eZ/Publish/Core/MVC/Symfony/View/BlockValueView.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\MVC\Symfony\View;
+
+use eZ\Publish\Core\FieldType\Page\Parts\Block;
+
+interface BlockValueView
```

Comment:
The idea is that a `BlockView` is both a `View` and View of a `Block` Value. Hence `BlockValueView`.